index.html

```diff
@@ -115,7 +115,28 @@ <h2>A project for first-time contributions</h2>
 
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
-              
+        <div class="card">
+          <p class="name">Indu Kushwaha</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/goforindu" target="_blank">@goforindu</a>
+          </p>
+          <p class="about">I am a frontend developer. I have experience in html5 css3 bootstrap4 javascript reactjs . I just want to restart my career . so I just learning to contribute to open-source projects. Hope this will help me and others also. so that I can share my learnings and skill with them. </p>
+          <div class="resources">
+            <p>3 Useful Dev Resources in my web development Journey</p>
+	           <ul>
+              <li>
+                <a href="https://www.w3schools.com" target="_blank" title="First Resource">w3schools</a>
+              </li>
+              <li>
+                <a href="https://www.udemy.com/course/react-the-complete-guide-incl-redux/" target="_blank" title="Second Resource">React course by maximilian Schwarzmuller </a>
+              </li>
+              <li>
+                <a href="https://stackoverflow.com/" target="_blank" title="Third resource">stackoverflow</a>
+              </li>
+            </ul>
+          </div>
+        </div>
         <!-- ________ Ze Xin's card START ________  -->
```

Comment:
add 
`<!-- ________ username card END ________ -->` and a line break after your card ends. Don't forget to replace the `username` with your name.